addarr.py

```diff
@@ -214,11 +257,30 @@ def searchSerieMovie(update, context):
     if searchResult:
         context.user_data["output"] = service.giveTitles(searchResult)
 
-        reply_keyboard = [
-            [transcript[choice.lower()]["Add"], transcript["Next result"]],
-            [transcript["New"], transcript["Stop"]],
-        ]
-        markup = ReplyKeyboardMarkup(reply_keyboard, one_time_keyboard=True)
+        keyboard = [
```

Comment:
I would make a 2x2 for this, instead of 1x4.

Reply:
I did that at first, but the strings won't fit inside the buttons and they just get cut off:
![image](https://user-images.githubusercontent.com/1628614/106911497-7f673d80-6702-11eb-9303-6f94375d11d3.png)

So I would propose to leave it 1x4 for now, and update the strings (and this format) in another PR, to prevent this PR becoming too big. Maybe add [i18n](https://pypi.org/project/python-i18n/) to simplify translations as well.